source/includes/cxi/_segments.md

```diff
@@ -0,0 +1,43 @@
+# Segments/Properties
+
+### HTTP Request
+
+`PUT https://cxi-api.wootric.com/v1/segments`
+
+## Response segments bulk update
+
+Parameter | Type | Description
+--------- | ---- | -------
+response[][id] | integer | Response ID
```

Comment:
Hahah I'm confused, is it plural or singular? 😅

Reply:
It's plural, thanks for all you comments